src/ddmd/doc.d

```diff
@@ -2405,6 +2412,32 @@ extern (C++) void highlightText(Scope* sc, Dsymbols* a, OutBuffer* buf, size_t o
                 }
             }
             break;
+
+        case '$':
```

Comment:
Would it be possible to reuse the `\xFF{` / `}` painting from dmacro.d here? This seems like it's duplicating the logic from there.